gke/variables.tf.json

```diff
@@ -17,7 +17,7 @@
         "location": {
             "type": "string",
             "description": "The GCP region where the cluster is placed, including the zone.",
-            "default": "us-central1-a"
+            "default": ""
```

Comment:
👍 

We should mention in the README that if the location is set to a region and not a specific zone GKE would set up a regional cluster (9 nodes) and not a zonal cluster (3 nodes), perhaps not something the operator wants.

https://www.terraform.io/docs/providers/google/r/container_cluster.html#argument-reference

Reply:
I removed that behavior. The number of nodes created is the number requested.

Reply:
This is somewhat orthogonal to the number of nodes requested. As the link explains, a regional cluster = 3 clusters with as many nodes requested per cluster so if someone asks for 5 nodes in their cluster and if they are not careful with their location selection they can end up with a regional cluster with 3 clusters of 5 nodes each.